docs/examples/ts/example.ts

```diff
@@ -0,0 +1,264 @@
+import {
+  OlpClientSettings,
+  VersionedLayerClient,
+  HRN,
+  DataRequest,
+  PartitionsRequest,
+  QuadKeyPartitionsRequest,
+  quadKeyFromMortonCode,
+  IndexLayerClient,
+  IndexQueryRequest,
+  StreamLayerClient,
+  SubscribeRequest,
+  PollRequest,
+  SeekRequest,
+  VolatileLayerClient
+} from "../../../@here/olp-sdk-dataservice-read";
+import {
+  UserAuth,
+  requestToken,
+  HttpError
+} from "../../../@here/olp-sdk-authentication/index.web";
```

Comment:
Please use npm packages and import from @here/olp-sdk-authentication

Reply:
Done